### PR TITLE
added "realization" to known coords

### PR DESCRIPTION
**Pull request by cehbrecht**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes issue #xyz
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)
- [ ] HISTORY.rst has been updated (with summary of main changes)
- [x] I have added my relevant user information to `AUTHORS.md`

* **What kind of change does this PR introduce?:** 

feature

* **Does this PR introduce a breaking change?:** 
no. 

But change is needed by rook version >=0.11.0

* **Other information:**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,4 +4,4 @@ xarray>=0.15
 dask[complete]>=2.6
 netCDF4>=1.4
 python-dateutil>=2.8.1
-cf-xarray>=0.3.1
+cf-xarray>=0.3.1,<=0.8.4
```
Comment (Zeitsperre) [inline requirements.txt:7]: This has repercussions downstream. As far as I can tell, the major issue is that cf_xarray is breaking in Python3.8. Could we add this upper pin only for a specific Python? I've started to do this for a few projects already.

File: environment.yml [hunk 2]
```diff
@@ -8,5 +8,5 @@ dependencies:
  - dask>=2.6
  - cftime
  - python-dateutil>=2.8.1
- - cf_xarray>=0.3.1 # Note that conda-forge package is cf_xarray (!= cf-xarray)
+ - cf_xarray>=0.3.1,<=0.8.4 # Note that conda-forge package is cf_xarray (!= cf-xarray)
```
Comment (Zeitsperre) [inline environment.yml:11]: See issue about cf_xarray in `requirements.txt`

File: requirements.txt (hunk 1 above)
Comment (Zeitsperre) [reply requirements.txt:7]:
```suggestion
cf-xarray>=0.3.1,<=0.8.4; python_version == '3.8'
cf-xarray>=0.3.1; python_version >= '3.9'
```

File: environment.yml (hunk 2 above)
Comment (Zeitsperre) [reply environment.yml:11]: Let's let conda figure this out on its own:
```suggestion
 - cf_xarray>=0.3.1 # Note that conda-forge package is cf_xarray (!= cf-xarray)
```

**Comment by cehbrecht** [comment]
@Zeitsperre thanks for review :) Ok to merge?

**Comment by cehbrecht** [comment]
@Zeitsperre I will do a release of roocs-utils. Next step would be to integrate roocs-utils as part of clisops ... reducing the number of packages we need to maintain.

**Review by Zeitsperre — APPROVED** [review]
Changes here look good, but before moving forward:
- The Python version classifiers in `setup.py` need to be updated
- The minimum Python version should be updated.

It'll be great to get this formally into CLISOPS!

**Pull request merged**